pkg/apis/autoscaling/types.go

```diff
@@ -95,6 +95,68 @@ type HorizontalPodAutoscalerSpec struct {
 	// more information about how each type of metric must respond.
 	// +optional
 	Metrics []MetricSpec
+
+	// Behavior contains settings to specify the scaling behavior both while scaling up
+	// and scaling down. It also is used to customize other aspects of scaling like the
+	// scale down stabilization window
+	// +option
+	Behavior *HorizontalPodAutoscalerBehavior
```

Comment:
Who uses this `pkg/apis/autoscaling/types.go` file?  All of the autoscaling imports I could find in the Kubernetes repo reference a versioned type.  E.g. `pkg/apis/autoscaling/v2beta2`.  Why the duplication?

Reply:
I'm not sure. We might need to add a ticket to investigate this and remove the duplication if possible.